stack-release.html

```diff
@@ -80,18 +80,9 @@
 						"SWC develop does not contain all patches to staging."
 					);
 
-					compareDiffAndUpdateElement(
```

Comment:
May want to move this down below (inside callback where we get package.json) and do this
```
compareDiffAndUpdateElement(
						"src_latest_tag_vs_main",
						"synapse-web-monorepo",
						`synapse-react-client/v${synapseReactClientLatestVersion}`,
						"main",
						"Latest synapse-react-client contains all changes in main.",
						"Latest synapse-react-client does not contain all changes in main."
					)
```

Or something like this, if this doesn't work exactly